### PR TITLE
SRCH-3647: health topic integration

### PR DESCRIPTION
**Pull request by yogesh27**
## Summary
- health topics integration
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji and explain why the step was not completed.
 
#### Functionality Checks
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
  
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket.

- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [x] Automated checks pass. If Code Climate checks do not pass, explain reason for failures:
 
#### Process Checks

- [x] You have specified at least one "Reviewer".

### CODE REVIEW

**Review by stevenbarragan — DISMISSED** [review]
LGTM

File: app/javascript/components/Results/Results.tsx [hunk 1]
```diff
@@ -71,6 +84,11 @@ export const Results = ({ query = '', results = null, additionalResults = null,
           
           {/* Health topics - To Do as part of backend integration */}
```
Comment (stevenbarragan) [inline app/javascript/components/Results/Results.tsx:85]: We should remove this comments. I don't think they're needed any more? right?

**Review by stevenbarragan — CHANGES_REQUESTED** [review]
One minor thing I didn't catch last time and we should be good to go

**Comment by yogesh27** [comment]
> One minor thing I didn't catch last time and we should be good to go

good catch

File: app/javascript/components/Results/Results.tsx [hunk 2]
```diff
@@ -69,8 +82,12 @@ export const Results = ({ query = '', results = null, additionalResults = null,
           {/* Jobs - To Do as part of backend integration */}
           {/* <Jobs /> */}
           
-          {/* Health topics - To Do as part of backend integration */}
           {/* <HealthTopics /> */}
```
Comment (stevenbarragan) [inline app/javascript/components/Results/Results.tsx:85]: Same comment as last time. This should be removed

**Review by stevenbarragan — CHANGES_REQUESTED** [review]

**Review by stevenbarragan — DISMISSED** [review]
LGTM 👍🏽

**Review by stevenbarragan — APPROVED** [review]
LGTM

**Pull request merged**